### PR TITLE
Catching must gather exception to avoid Runtime error

### PR DESCRIPTION
**Pull request by lalon4**
example: https://auto-jenkins-csb-kniqe.apps.ocp4.prod.psi.redhat.com/view/Assisted-Installer/job/ocp-assisted-installer-virt/223/testReport/junit/test_disk_selection/TestDiskSelection/test_remove_disk_boot_flag_2/

### CODE REVIEW

File: discovery-infra/download_logs.py [hunk 1]
```diff
@@ -164,7 +164,10 @@ def download_logs(client: InventoryClient, cluster: dict, dest: str, must_gather
                 recreate_folder(os.path.join(output_folder, "must-gather"))
                 config_etc_hosts(cluster['name'], cluster['base_dns_domain'],
                                  helper_cluster.get_api_vip_from_cluster(client, cluster))
-                download_must_gather(kubeconfig_path, os.path.join(output_folder, "must-gather"))
+                try:
```
Comment (slaviered) [inline discovery-infra/download_logs.py:167]: don't catch it here but inside download_must_gather

Comment (lalon4) [reply discovery-infra/download_logs.py]: OK, done

**Pull request merged**